lib/util.rb

```diff
@@ -206,39 +206,54 @@ def self.get_data(project, page, per_page, res)
     resource_data
   end
 
-  # this method takes a project name and returns a nice hash of all its
+  # this method takes a list of project ids and returns a nice hash of all their
   # resources and their measurments
-  def self.project_data(project, start_date, end_date)
+  def self.project_data(project_ids,
+                        start_date = Time.now - 2_592_000,
+                        end_date = Time.now)
     project_data = {}
 
-    # for each resource type in the matrix, get a list of all that type
-    # of resource each project has
-    plugin_matrix.each do |resource_type, measurements|
-      resource_data = {}
-      resources = project.send("#{resource_type}_resources")
-      next if resources.nil?
-      # for each of those resources, get all the measuremnts for that
-      # type of resource. Put it all in a big hash.
-      resources.each do |resource|
-        next unless resource.active
-        resource_data[resource.name] ||= {}
-        resource_data[resource.name]['id'] = resource[:id]
-        measurements.each do |measurement|
-          plugin = Object.const_get(measurement).new
-          data = plugin.report({ resource_type.to_sym => resource.name },
-                               start_date, end_date)
-          data_average = if data[0].nil?
-                           measurement == 'DiskTemplate' ? 'N/A' : 0
-                         elsif data[0][:value].number?
-                           DataUtil.average_value(data)
-                         else
-                           data[-1][:value]
-                         end
-          data_average = DataUtil.unit_conversion(measurement, data_average)
-          resource_data[resource.name].merge!(measurement => data_average)
+    plugin_tables = Plugin.select_map([:name, :resource_name, :storage_table])
+
+    plugin_tables.each do |row|
+      resource_key = (row[1] + '_resource').to_sym
+      resources_table = (row[1] + '_resources').to_sym
+      project_data[row[1]] ||= {}
+
+      data = Iam.settings.DB[resources_table].where(project_id: project_ids)
```

Comment:
I'm getting 
```
SQLite3::SQLException: no such table: resource_resources
```

I printed out plugin_tables and noticed the testingPlugin is included:
```
TestingPlugin
resource
test_plugin_measurements
```

is my migrations file wrong or is the code not suppose to parse the TestingPlugin?